packages/components/psammead-navigation/README.md

```diff
@@ -40,7 +40,37 @@ The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl
 | currentPageText | string | No | `null`  | `Current page` |
 | service | string | Yes | N/A | `'news'` |
 
-## Usage
+### CanonicalDropdown
```

Comment:
There should be an amp example of this and the hamburger since the `onClose` for amp is a string so slightly different.